lms/validation/_lti_launch_params.py

```diff
@@ -154,6 +154,22 @@ class DeepLinkingLTILaunchSchema(_CommonLTILaunchSchema):
     content_item_return_url = fields.Str(required=True)
 
 
+class AutoGradingConfigSchema(Schema):
```

Comment:
Moved here from:

lms/views/lti/deep_linking.py

this is now common to other type of launches.